10-Sprint-Mission/index.html

```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="./css/style.css">
+  <title>판다마켓</title>
+</head>
+<body>
+  <header class="header">
+    <a href="index.html">
+      <img src="./img/lg/Property 1=md.png" alt="로그인">
```

Comment:
P1:
이미지 명에 공백은 들어가지 않는 것이 좋습니다.
또한 어떤 이미지인지 알 수 있는 이름(logo.png)이면 좋을 것 같습니다.

```suggestion
      <img src="./img/logo.png" alt="판다마켓 로고">
```